dask/dataframe/tests/test_groupby.py

```diff
@@ -2094,6 +2094,32 @@ def test_series_groupby_idxmax_skipna(skipna):
     assert_eq(result_pd, result_dd)
 
 
+def test_groupby_unique():
+    rng = np.random.RandomState(42)
+    df = pd.DataFrame(
+        {"foo": rng.randint(3, size=100), "bar": rng.randint(10, size=100)}
+    )
+    ddf = dd.from_pandas(df, npartitions=10)
+
+    pd_gb = df.groupby("foo")["bar"].unique()
+    dd_gb = ddf.groupby("foo")["bar"].unique()
+
+    # Use explode because each DataFrame row is a list; equality fails
+    assert_eq(dd_gb.explode(), pd_gb.explode())
```

Comment:
I see the issue now. dask.dataframe.explode will only work if the pandas version has it, i.e. pandas 0.25.0. But we also support older pandas.

So this test needs to be skipped on pandas<0.25.0, or rewritten to not use explode. Skipping is probably easiest.

Reply:
Resolved. I skip the test is Panda's version is less than 0.25.